docker/tensorflow/mivisionx-with-tensorflow.dockerfile

```diff
@@ -0,0 +1,57 @@
+FROM rocm/tensorflow:latest
+
```

Comment:
@kiritigowda Can be ignored I guess? We  use latest tag for all other dockerfiles as well?